lib/checkout.rb

```diff
@@ -27,8 +37,19 @@ def self.marc_value(record, marc, subfield)
     nil
   end
 
+  def self.map_item_types_to_coarse_item_types(item_type)
+    @@item_types[item_type.to_i]
+  end
+
+  def self.circulating?(item_type)
```

Comment:
Should this be named "collection_type" or something? `circulating?` suggests to me it returns a boolean that is true if it's a circulating item.

Reply:
Fixed!